client/src/core/components/App.tsx

```diff
@@ -12,15 +12,21 @@ declare var API_HOST: string;
 
 const authToken = localStorage.getItem('token');
 
+const Loader = () => (
+    <div>Loading...</div>
+)
+
 const App: React.FC = (): JSX.Element => (
     <div>
         <ApolloProvider client={createApolloClient(API_HOST, authToken)}>
             <Router>
-                <NavBar />
-                <div className="site-content">
-                    <InitialSubmissionRoutes />
-                    <DashboardRoutes />
-                </div>
+                <React.Suspense fallback={<Loader />}>
```

Comment:
Is this not currently only used for suspending rendering of dynamically imported assets until they're fully loaded in? Or is this to allow the translation stuff to load something?

Reply:
I think its both